Formula/px4-sim-gazebo-classic.rb

```diff
@@ -0,0 +1,39 @@
+class XRequirement < Requirement
+  fatal true
+
+  satisfy(build_env: false) { which("xquartz") }
+
+  def message
+    <<~EOS
+      XQuartz is required; install it via:
+        brew install --cask xquartz
+    EOS
+  end
+end
+
+class Px4SimGazeboClassic < Formula
+  desc "PX4 Gazebo simulation"
+  homepage "http://px4.io"
+  url "https://raw.githubusercontent.com/PX4/PX4-Autopilot/master/Tools/px4.py"
```

Comment:
Since this file is new, please use the updated URL

https://raw.githubusercontent.com/PX4/PX4-Autopilot/refs/heads/main/Tools/px4.py

Reply:
huh ok. replaced the link.